virttest/utils_time.py

```diff
@@ -0,0 +1,248 @@
+import logging
+import re
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+from virttest import error_context
+from virttest import utils_disk
+
+
+def get_host_timezone():
+    """
+    Get host's timezone.
+    """
+    timezone_cmd = 'timedatectl | grep "Time zone"'
+    timezone_pattern = '^(?:\s+Time zone:\s)(\w+\/\S+|UTC)(?:\s\(\S+,\s)([+|-]\d{4})\)$'
+    error_context.context("Get host's timezone", logging.info)
+    host_timezone = process.system_output(timezone_cmd, timeout=240, shell=True)
+    try:
+        host_timezone_set = re.match(timezone_pattern, host_timezone).groups()
+        return {"timezone_city": host_timezone_set[0],
+                "timezone_code": host_timezone_set[1]}
+    except AttributeError, IndexError:
+        raise exceptions.TestError("Fail to get host's timezone.")
+
+
+def verify_timezone(session):
+    """
+    Verify linux guest's timezone.
+
+    :param session: VM session.
+    """
+    error_context.context("Verify guest's timezone", logging.info)
+    timezone_cmd = 'timedatectl | grep "Time zone"'
+    timezone_pattern = '^(?:\s+Time zone:\s)(\w+\/\S+|UTC)(?:\s\(\S+,\s)([+|-]\d{4})\)$'
+    guest_timezone = session.cmd_output(timezone_cmd, timeout=240)
+    try:
+        guest_timezone_set = re.match(timezone_pattern, guest_timezone).groups()
+        return (guest_timezone_set[0] == get_host_timezone()['timezone_city'])
+    except AttributeError, IndexError:
+        raise exceptions.TestError("Fail to get guest's timezone.")
+
+
+def sync_timezone(session):
```

Comment:
If it only works for linux, would it be better to name it like `sync_timezone_linux()`? the same for `verify_timezone()`

Reply:
Thanks, updated it.